src/mmda/types/box.py

```diff
@@ -95,11 +116,6 @@ def coordinates(self) -> Tuple[float, float, float, float]:
     def center(self) -> Tuple[float, float]:
         return self.l + self.w / 2, self.t + self.h / 2
 
-    @property
-    def xywh(self) -> Tuple[float, float, float, float]:
-        """Return a tuple of the (left, top, width, height) format."""
-        return self.l, self.t, self.w, self.h
```

Comment:
Let's leave the method in but add a logging.warning() with a deprecation notice. This may break clients of mmda that use `.xywh`